src/app/components/search-bar/search-bar.component.ts

```
@@ -169,6 +176,21 @@ export class SearchBarComponent implements OnInit, OnChanges, OnDestroy {
         this.loadRegions(this.selectedRegions);
+
+        const vendorRegionValues = this.query.vendor_regions;
+        if (vendorRegionValues) {
+          const values: string[] = Array.isArray(vendorRegionValues)
+            ? vendorRegionValues
+            : [vendorRegionValues];
+          [
+            ...new Set(
+              values.map((vr: string) => vr.split("~")[0]).filter(Boolean),
+            ),
+          ].forEach(
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)